project.clj

```diff
@@ -78,7 +78,8 @@
                  [puppetlabs/trapperkeeper ~tk-version]
                  [puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty9-version]
                  [prismatic/schema "1.0.4"]
-                 [puppetlabs/trapperkeeper-status ~tk-status-version :exclusions [trptcolin/versioneer]]
+                 [puppetlabs/trapperkeeper-status ~tk-status-version]
+                 [trptcolin/versioneer "0.2.0"]
```

Comment:
might want to put this above the status service since it's a dependency of status